Sources/MapboxMaps/Ornaments/ScaleBar/MapboxScaleBarOrnamentView.swift

```diff
@@ -43,25 +48,26 @@ internal class MapboxScaleBarOrnamentView: UIView {
             var bars: [UIView] = []
             for _ in 0..<row.numberOfBars {
                 let bar = UIView()
-                self.containerView.addSubview(bar)
+                self.dynamicContainerView.addSubview(bar)
                 bars.append(bar)
             }
             _bars = bars
         }
         return _bars!
     }
 
-    lazy private var containerView: UIView = {
+    var size = CGSize()
+    // This container view's size and position can change based on the size
+    // of its contents. It is contained within the `staticContainerView`.
+    lazy private var dynamicContainerView: UIView = {
         let view = UIView()
-        view.clipsToBounds = true
+//        view.clipsToBounds = true
```

Comment:
we shouldn't use commented out code on the final commit

Reply:
Agreed, I'm still debugging this code.